### ISSUE TITLE
Sequoia 15 Beta 8 not installing generated packages

### ISSUE THREAD

**Issue by dvdborn** [gregneagle/pycreateuserpkg#70]
Generated packages, even when they have been signed, can't be installed by Sequoia 15 Beta 8.

When you try to install a package you'll get an error window telling you that the package can't be installed.
![pycreateuserpkg01](https://github.com/user-attachments/assets/47f69c3d-bad8-4063-ac27-dab3ed07bd1a)

This is what's being logged by the Installer.app:
![pycreateuserpkg02](https://github.com/user-attachments/assets/986962a1-5ad3-429e-a088-ffd5ee3df6ae)

It looks like the createuser executable in the installer package also needs to be signed in order to perform the installation.
As a workaround I expanded the generated package using pkgutil. I then tried to sign createuser using the productsign command. However this wasn't possible (error: Can't read input product archive "./createuser").

**Comment by gregneagle** [comment]
`productsign` is for signing packages; you need to use `codesign` to sign executables.

**Comment by dvdborn** [comment]
Thanks, my mistake.

I was wrong in thinking that the issue was with pycreateuserpkg, and more specificaly with the createuser executable. 

However, it's Gatekeeper that's preventing to execute createuser. Only codesigning the executable isn't enough to fix this. It needs to be notorized by Apple in order to pass Gatekeeper. With Gatekeeper disabled the pycreateuserpkg generated package installed without a problem.

In short, there's no issue with pycreateuserpkg under macOS Sequoia.
